index.html

```diff
@@ -19,7 +19,7 @@
 
       // the specification's short name, as in
       // http://www.w3.org/TR/short-name/
-      shortName: "vc-jwt",
+      shortName: "vc-jose-cose",
```

Comment:
could you elaborate a little more why not `vc-jose`, but `vc-jose-cose`? what is the plan for cose?

Reply:
We define how to secure the media types registered in https://w3c.github.io/vc-data-model/

We already explain how to secure them as COSE Sign1.

https://w3c.github.io/vc-jwt/#with-cose

We don't want key discovery or other header processing details to be different for JOSE and COSE... we want consistent alignment between protected header values and key and algorithms.

for example:

In jose, protected header contains `alg` and `kid`... `kid` used to discover `jwk` with same `alg`.... 
In cose, protected header contains `alg` and `kid`... `kid` used to discover `cose key` with same `alg`.... 

When `iss` is present in the header, is `kid` relative... see: https://identity.foundation/jwt-vc-presentation-profile/#kid-jose-header

We need to address these usability issues related to W3C specs, consistently for JOSE and COSE.